buildspec.yml

```diff
@@ -46,21 +46,11 @@ phases:
         tox -e py27,py36,stats -- tests/
 
       # spark integration tests
-      - cd $CODEBUILD_SRC_DIR/sagemaker-spark-sdk
-      - |
-        if has-matching-changes "src/" "test/" "build.sbt"; then
-          cd $CODEBUILD_SRC_DIR/integration-tests/sagemaker-spark-sdk
-          sbt -Dsbt.log.noformat=true it:test
-        else
-          echo "skipping spark integration tests"
-        fi
+      - cd $CODEBUILD_SRC_DIR/integration-tests/sagemaker-spark-sdk
+      - test_cmd="sbt -Dsbt.log.noformat=true it:test"
+      - execute-command-if-has-matching-changes "$test_cmd" "src/" "test/" "build.sbt" "buildspec.yml"
 
       # pyspark integration tests
       - cd $CODEBUILD_SRC_DIR/sagemaker-pyspark-sdk
-      - |
-        if has-matching-changes "src/" "tests/" "setup.*" "requirements.txt" "tox.ini"; then
-          IGNORE_COVERAGE=- \
-          tox -e py36 -- $CODEBUILD_SRC_DIR/integration-tests/sagemaker-pyspark-sdk/tests/ -n 10 --boxed --reruns 2
-        else
-          echo "skipping pyspark integration tests"
-        fi
+      - test_cmd="IGNORE_COVERAGE=- && tox -e py36 -- $CODEBUILD_SRC_DIR/integration-tests/sagemaker-pyspark-sdk/tests/ -n 10 --boxed --reruns 2"
```

Comment:
you don't need/want the `&&` - the point is to have `IGNORE_COVERAGE` set during the `tox` command

Reply:
```suggestion
      - test_cmd="IGNORE_COVERAGE=- tox -e py36 -- $CODEBUILD_SRC_DIR/integration-tests/sagemaker-pyspark-sdk/tests/ -n 10 --boxed --reruns 2"
```